### ISSUE TITLE
Write contributor guide

### ISSUE THREAD

**Issue by ewallace** [riboviz/RiboViz#459]
@lianafaye asked about best way to contribute new features to riboviz

We should write a contributor guide for that!

**Comment by FlicAnderson** [comment]
I agree, this is a great idea and would be worth doing.  Possibly another good task for @B209678-2021 and @Maberuiz once they've gotten further into their projects and are more confident developing and working with riboviz and adding new datasets? @ewallace what do you think? 

There's a recommended set of things github think projects should have to better engage with their communities: https://github.com/riboviz/riboviz/community

I think this issue would tick one off, and might also tie in nicely with BBSRC-NSF issue [41](https://github.com/riboviz/bbsrc-nsf/issues/41) regarding community engagement.
